lib/Importer/ONNXModelLoader.cpp

```diff
@@ -36,14 +36,29 @@ using namespace glow;
 using llvm::cast;
 
 namespace {
+
+/// Get the shape of a TensorShapeProto given by \p shapeProto and return the
+/// dimensions in the vector \p dim passed by reference.
+Error getProtoShape(const ONNX_NAMESPACE::TensorShapeProto &shapeProto,
```

Comment:
tiny nit: Could return `Expected<std::vector<size_t>>` here instead of `Error`.

Reply:
Done.